### PR TITLE
Final fixes

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-cmgg/exomecnv/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core pipelines lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/wes.config [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running the pipeline on the CMGG infrastructure
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
+params {
+    exomedepth = true
+    annotate = true
+    roi_auto = "${params.genomes_base}/Hsapiens/GRCh38/regions/Homo_sapiens.GRCh38.105.chr_protein_coding_basic_sorted_merged_autosomal.bed"
+    roi_chrx = "${params.genomes_base}/Hsapiens/GRCh38/regions/Homo_sapiens.GRCh38.105.chrX_protein_coding_basic_sorted_merged.bed"
```
Comment (matthdsm) [inline conf/wes.config:11]: also add vep_cache here?

File: modules/nf-core/ensemblvep/vep/ensemblvep-vep.diff [hunk 2]
```diff
@@ -1,22 +1,29 @@
...
-'modules/nf-core/ensemblvep/vep/meta.yml' is unchanged
 Changes in 'ensemblvep/vep/main.nf':
 --- modules/nf-core/ensemblvep/vep/main.nf
 +++ modules/nf-core/ensemblvep/vep/main.nf
-@@ -1,6 +1,6 @@
+@@ -1,11 +1,11 @@
  process ENSEMBLVEP_VEP {
      tag "$meta.id"
 -    label 'process_medium'
 +    label 'process_low'
```
Comment (matthdsm) [inline modules/nf-core/ensemblvep/vep/ensemblvep-vep.diff:9]: why process_low?

File: conf/wes.config [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+}
```
Comment (ToonRosseel) [inline conf/wes.config:12]: also add the VEP cache location here?

File: conf/wes.config (hunk 1 above)
Comment (nvnieuwk) [reply conf/wes.config:11]: It's in the `main.nf` and will be fetched from the genomes config :)

File: modules/nf-core/ensemblvep/vep/ensemblvep-vep.diff (hunk 2 above)
Comment (nvnieuwk) [reply modules/nf-core/ensemblvep/vep/ensemblvep-vep.diff:9]: @ToonRosseel ?

Comment (ToonRosseel) [reply modules/nf-core/ensemblvep/vep/ensemblvep-vep.diff:9]: don't remember anymore why I changed the default. my guess: only +-200-300 variants per sample so no high memory is needed. but it's ok for me to revert it to "medium"

Comment (ToonRosseel) [reply modules/nf-core/ensemblvep/vep/ensemblvep-vep.diff:9]: in current setting, 4 cpus are used for the annotation step

Comment (nvnieuwk) [reply modules/nf-core/ensemblvep/vep/ensemblvep-vep.diff:9]: That seemed to work perfectly on the test run

Comment (ToonRosseel) [reply modules/nf-core/ensemblvep/vep/ensemblvep-vep.diff:9]: if you agree @matthdsm, I would leave it "low" for faster process planning

**Pull request merged**